docs/conf.py

```diff
@@ -91,12 +92,22 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-#
-html_theme = "sphinx_rtd_theme"
+
+html_theme = 'sphinx_wagtail_theme'
 html_logo = "figures/logo.svg"
 html_favicon = "figures/favicon.ico"
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = []
+html_static_path = ["_static"]
+html_css_files = ["custom.css"]
+
+# This github_url will point to the appropriate page in the default branch.
+# Ex: Getting Started -> https://github.com/vprusso/toqito/blob/master/docs/getting_started.rst
+html_theme_options = dict(
+    github_url = "https://github.com/vprusso/toqito/blob/master/docs/"
+)
```

Comment:
Why is `https://github.com/vprusso/toqito/blob/master/docs/` used here? Why not `https://github.com/vprusso/toqito` ?

Reply:
The current `Edit on GitHub` button on the docs brings you to: https://github.com/vprusso/toqito/blob/master/docs/index.rst

Reply:
The current `Edit on Github` takes you to whatever file is being used to generate the documentation. 

That's what we see linked by the  `Edit on Github` for the home page, we see another file for the intro tutorial, and so on.

I wonder if we could link to `https://github.com/vprusso/toqito` when the same button links to a file that does not exist in `API Reference`. I don't know if this is possible but just throwing it out as a suggestion. Helps us avoid adding a bunch of files whenever sphinx is used to build the docs.

Reply:
That would be great, thanks!